src/commands/aem/rde/install.js

```diff
@@ -224,8 +252,12 @@ function guessType(name, url, pathFlag) {
         if (isDispatcherConfig) {
           return ['dispatcher-config'];
         }
+        let isFrontendPipeline = zip.getEntry('dist/') !== null && zip.getEntry('package.json') !== null
+        if (isFrontendPipeline) {
+          return ['frontend-pipeline']
+        }
       }
-      return ['content-package', 'dispatcher-config'];
+      return ['content-package', 'dispatcher-config', 'frontend-pipeline'];
```

Comment:
imho `frontend-pipeline` is a confusing name for a packaging format. Shouldn't it rather be `static-frontend-directory` or something along those lines?